presto-product-tests/src/main/java/com/facebook/presto/tests/convention/TestShowPartitions.java

```diff
@@ -68,6 +84,19 @@ private static TableDefinition partitionedTableDefinition()
                 .build();
     }
 
+    private static TableDefinition partitionedTableWithVariablePartitionsDefinition()
```

Comment:
it is not `Variable` any more. Rename this method to `partitionedTableWithTooManyPartitionsDefinition`, please to the same with `PARTITIONED_TABLE_WITH_VARIABLE_PARTITIONS`

Reply:
It is variable. Every test (currently 1) that uses this table is responsible for populating it with partitions.

Reply:
Also, I cannot rename it to `...WithTooManyPartitions...`, because the table (as is defined here) does not have too many partitions (it's empty).

Reply:
It is agains the tempto conventions. As you could see in other tests, we try to avoid initialization code in tests. No CREATE TABLE, INSERT etc. This is provided by tempto as fulfilment of test requirements. 

IMO you should have tables with fixed number of partitions. If you need tables with 99, 100 and 101 partitions, you should have 3 different requirements definied.

Reply:
I know this is against tempto conventions, but I have a good reason to depart from them here: tempto doesn't support immutable partitioned tables (see TODO in the code), so I would pay the initialization cost per every test method.